docs/en/build-store/manage.md

```diff
@@ -8,4 +8,4 @@ table_of_contents: true
 Brand stores have two administrative roles: admins and reviewers.
 
 * Admins manage general settings of the store, such as security policies enforced on packages and snaps it includes from other stores.<br>[What are the available settings?](settings.md)
-* Reviewers control the flow of packages to users, comment on, approve or reject uploaded snaps, grant per-snap overrides for warnings and errors detected by the automated review process and set up per-snap features, such as automated aliases and auto-connecting interfaces.<br>[How to review a snap?](reviews.md)
+* Reviewers control the flow of packages to users, comment on, approve or reject uploaded snaps, grant per-snap overrides for warnings and errors detected by the automated review process and set up per-snap features, such as automated aliases, auto-connecting interfaces, and refresh control.<br>[How to review a snap?](reviews.md)
```

Comment:
I suppose Refresh Control is a product, a feature. As such it should always be capitalized.

Reply:
Good point! Done.